asmcli/asmcli.sh

```diff
@@ -2061,7 +2061,8 @@ register_cluster() {
   else
     CMD="${CMD} --kubeconfig=${KCF} --context=${KCC}"
   fi
-  retry 2 "${CMD}"
+  # shellcheck disable=SC2086
+  retry 2 ${CMD}
```

Comment:
Can you also add a `run_command` between 2 and ${CMD}?

Reply:
done.